cockatrice/src/zoneviewwidget.cpp

```diff
@@ -89,6 +89,11 @@ ZoneViewWidget::ZoneViewWidget(Player *_player, CardZone *_origZone, int numberC
         QGraphicsProxyWidget *sortByTypeProxy = new QGraphicsProxyWidget;
         sortByTypeProxy->setWidget(sortByTypeCheckBox);
         vbox->addItem(sortByTypeProxy);
+
+        pileViewCheckBox = new QCheckBox;
```

Comment:
Agreed.
There are lots of files I want to clean up. I kept with the convension for this PR, I will add another that refactors the entire code very soon.